src/openMVG/matching_image_collection/GPU_Matcher_Regions.cpp

```diff
@@ -0,0 +1,140 @@
+
```

Comment:
Please consider to use the new updated header
https://github.com/openMVG/openMVG/blob/develop/src/openMVG/image/image_container.hpp#L1
Add `
// This file is part of OpenMVG, an Open Multiple View Geometry C++ library.`